### PR TITLE
Visual fixes, spinners

### PR DESCRIPTION
**Pull request by or-else**
* Changed the tags icon to be more like other icons (glyph vs outline).
* Fixed tint not applied to some icons.
* Show progress spinner in LoginVC.
* Add missing `super` calls for some overriden methods.

### CODE REVIEW

File: Tinodios/UiUtils.swift [hunk 1]
```diff
@@ -311,6 +311,22 @@ class UiUtils {
...
+
+            let loadingIndicator = UIActivityIndicatorView(frame: CGRect(x: 10, y: 5, width: 50, height: 50))
+            loadingIndicator.hidesWhenStopped = true
+            loadingIndicator.style = UIActivityIndicatorView.Style.gray
+            loadingIndicator.startAnimating();
+
+            alert.view.addSubview(loadingIndicator)
+            parent.present(alert, animated: true, completion: nil)
+        } else if let vc = parent.presentedViewController, vc is UIAlertController {
+            parent.dismiss(animated: false, completion: nil)
```
Comment (aforge) [inline Tinodios/UiUtils.swift]: Should dismiss be called on vc and not parent?

File: Tinodios/UiUtils.swift [hunk 2]
```diff
@@ -311,6 +311,22 @@ class UiUtils {
...
     }
+
+    public static func toggleProgressOverlay(in parent: UIViewController, visible: Bool, title: String? = nil) {
+        if visible {
+            let alert = UIAlertController(title: nil, message: title ?? "Please wait...", preferredStyle: .alert)
+
+            let loadingIndicator = UIActivityIndicatorView(frame: CGRect(x: 10, y: 5, width: 50, height: 50))
+            loadingIndicator.hidesWhenStopped = true
+            loadingIndicator.style = UIActivityIndicatorView.Style.gray
+            loadingIndicator.startAnimating();
```
Comment (aforge) [inline Tinodios/UiUtils.swift]: Remove semicolon please.

File: Tinodios/UiUtils.swift (hunk 1 above)
Comment (or-else) [reply Tinodios/UiUtils.swift]: Calling on the parent is the way to go:
https://developer.apple.com/documentation/uikit/uiviewcontroller/1621505-dismiss
> `dismiss(animated:completion:)`
> Dismisses the view controller that was presented modally by the view controller.

It works as expected. Try entering a wrong password.

It **can** be called on the alert VC though:
> If you call this method on the presented view controller itself, UIKit asks the presenting view controller to handle the dismissal.

File: Tinodios/UiUtils.swift (hunk 2 above)
Comment (or-else) [reply Tinodios/UiUtils.swift]: done here and in a few other places.

File: Tinodios/UiUtils.swift (hunk 1 above)
Comment (aforge) [reply Tinodios/UiUtils.swift]: Nice! Thanks!

**Pull request merged**